core/src/main/java/com/novoda/noplayer/player/ExoPlayerCapabilities.java

```diff
@@ -1,14 +1,21 @@
 package com.novoda.noplayer.player;
 
-import com.novoda.noplayer.drm.DrmHandler;
-import com.novoda.noplayer.drm.DownloadedModularDrm;
-import com.novoda.noplayer.drm.StreamingModularDrm;
+import com.novoda.noplayer.drm.DrmType;
+
+import java.util.Arrays;
+import java.util.List;
 
 class ExoPlayerCapabilities implements PlayerCapabilities {
 
+    private static final List<DrmType> SUPPORTED_DRM_TYPES = Arrays.asList(
```

Comment:
I would still not unless strong objection. For me, the intention should be clear from the name and `static final` and wrapping it just to have it crash seems excessive, _especially_ as we're not exposing it.